components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -91,19 +92,17 @@ export class TeamManagementComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.store.dispatch(new GetTeams());
 
-    this.iamMajorVersion$
+    this.store.select(iamMajorVersion)
       .pipe(takeUntil(this.isDestroyed))
       .subscribe((majorVersion) => {
-        if (majorVersion === null) { return; }
-          this.isMajorV1 = majorVersion === 'v1';
-        });
```

Comment:
❓ Should this become a `filter(identity)` in the pipe, or is it fine to just drop it?

Reply:
My feeling is that the chance of being null here is very small but even if it was, it would only be for a very short time.